pages/wishlist.html

```diff
@@ -0,0 +1,282 @@
+      rel="stylesheet"
+      href="https://powertrain-ui.netlify.app/styles.css"
+    />
+
+    <!-- Page CSS -->
+    <link rel="stylesheet" href="../css/wishlist.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <div class="header pos-st top-left-pos py-md">
+        <header class="max-width-1200 nav-main flex-row px-md mx-auto">
+          <div class="nav-logo">
+            <a href="../../index.html">FueledUp</a>
+          </div>
+          <div class="search-input">
+            <div class="fas fa-search search-icon"></div>
+            <label for="searchbar"></label>
+            <input
+              class="input-round input-md border-2"
+              type="search"
+              name="searchbar"
+              id="searchbar"
+              placeholder="Search products..."
+            />
+          </div>
+          <button class="hamburger">
+            <i class="fas fa-bars"></i>
+          </button>
+          <nav class="nav-list">
+            <button class="nav-list-close-btn">
+              <i class="fas fa-2x fa-times"></i>
+            </button>
+            <ul class="nav-menu h-100">
+              <li class="nav-item">
+                <a href="../../index.html"
+                  ><button class="btn btn-md btn-outline px-md">
+                    Logout
+                  </button></a
+                >
+              </li>
+              <li class="nav-item">
+                <div class="badge">
+                  <div class="fas fa-heart badge-icon txt-md px-sm"></div>
+                  <div class="badge-count top-right flex-center badge-yellow">
+                    6
+                  </div>
+                  <a href="#" class="ecom-heart-icon"></a>
+                </div>
+              </li>
+              <li class="nav-item">
+                <div class="badge">
+                  <div
+                    class="fas fa-shopping-cart badge-icon txt-md px-sm"
+                  ></div>
+                  <div class="badge-count top-right flex-center badge-yellow">
+                    2
+                  </div>
+                  <a href="#" class="ecom-basket-icon"></a>
+                </div>
+              </li>
+            </ul>
+          </nav>
+        </header>
+        <div class="mob-search-input hide">
+          <div class="fas fa-search mob-search-icon"></div>
+          <label for="searchbar"></label>
+          <input
+            class="input-round input-md border-2"
+            type="search"
+            name="searchbar"
+            id="searchbar"
+            placeholder="Search products..."
+          />
+        </div>
+      </div>
+      <div class="main my-2">
+        <main class="max-width-1200 mx-auto">
+          <h2 class="page-title txt-bold mx-auto py-md">My Wishlist</h2>
+          <hr />
+          <section
+            class="page-content grid grid-cols-auto grid-col-1 grid-cols-2 gap-1 mx-auto px-lg"
+          >
+            <div class="card mx-auto">
```

Comment:
would suggest using semantic for the card layout if  you are not using the component library cards and made this specifically for ecom screens.

Reply:
I have used it from component library, have tweaked it where specifically needed for ecommerce
Overall code does follow semantic practices to the best of my knowledge